docs/user/troubleshooting/plasma.md

```diff
@@ -0,0 +1,99 @@
+- Mouse cursor is stuck in move mode, it looks like a cross.
+
+You may not need to reboot to recover from this state. Killing all firefox processes may be all that's necessary. To do so:
+
+1. Open a virtual terminal by pressing `Ctrl+Alt+F3`.
+1. Run the following to kill all firefox windows:
+  ```bash
+  killall firefox
+  ```
+  This should close all Firefox windows.
+1. Return to the Plasma session by pressing `Ctrl-Alt-F2`.
+  You should now be able to use Plasma normally.
+
+## Application windows look corrupt / incomplete
+
+**Symptoms**
+- Application windows look corrupt. For example, NeoChat does not display room or user icons.
+- Menus in an application are blank.
+
+You may be able to fix this by clearing KDE / Plasma cache files, by following the steps below.
+
+:::caution
+
+You should not be logged into your Plasma session while doing this. It is also highly recommended to back up the folders mentioned below before deleting anything.
+
+:::
+
+1. While still logged into your Plasma session, cd to your home directory:
+  ```bash
+  cd
+  ```
+1. Run these commands to back up the following directories:
+  ```bash
+  cp -av ~/.kde{,.bak}
+  cp -av ~/.config{,.bak}
+  cp -av ~/.local{,.bak}
+  cp -av ~/share{,.bak}
+  ```
+  The output will show what files were copied. Each directory will have another copy with a .bak extension such as `~/.config.bak`. These can be used to recover configuration files, if necessary.
+1. Shut down the system.
+1. Boot from a live USB, which you can create from one of the [Solus installers](https://getsol.us/download/).
+2. [Mount your hard drive's file system](./boot-rescue.md#mounting-your-system).
+3. Change to your user's directory where you mounted your file system. For example, if you used `/mount`:
+  ```bash
+  cd /mount/home/username
+  ```
+  Replace "username" with the username you use for Solus.
+  You can make sure the directory is correct by listing the files:
+  ```bash
+  ls -laH
+  ```
+  You should see the files you would usually see in your home directory when logged into Solus normally.
+1. Now you need to remove the various caches that Plasma uses.
+  Execute the following commands:
+  ```bash
+  rm -rf .kde
+  rm -rf .cache/plasmashell*
+  rm -rf .cache/org.kde.dirmodel-qml.kcache
+  rm -rf .cache/kioexec/ .cache/krunner/ .cache/ksycoca5*
+  rm -rf .cache/krunnerbookmarkrunnerfirefoxdbfile.sqlite
+
+  cd ../.local/
+  rm -rf kate/ kded5/ klipper/ knewstuff3/ kscreen/ konsole/ kwalletd/ ksysguard/ kmail2/ kcookiejar/ kactivitymanagerd/
+
+  cd ../share/
+  rm -rf dolphin kate kcookiejar kded5 keyrings klipper kmail2 knewstuff3 konsole kscreen ksysguard kwalletd kxmlgui5 plasma_engine_comic plasma plasma_notes org.kde.gwenview
+
+  cd ../.config/
+  rm -f plasma*
+  rm -rf kde* akonadi* KDE kconf_updaterc baloo* dolphinrc drkonqirc gwenviewrc kmail2rc k*rc katemetainfos
+  ```
+1. Shut down the live environment and remove the live USB drive.
```

Comment:
Does it matter that a lot of steps are marked as "1."? Or Docusaurus will render everything properly?

Reply:
Docusaurus renders the numbering correctly. You can see this if you check out this branch and run `yarn start`

Since that's the case, it's actually preferable to use 1. for all numbered lines to prevent having to manually number them.